test/events/component-view.test.js

```diff
@@ -0,0 +1,208 @@
+/*global require, describe, it, before, after, sinon, document */
+require('../setup');
+const assert = require("assert");
+const settings = require("../../src/javascript/core/settings");
```

Comment:
These imports typically use single quotes

Reply:
Oh thank you!